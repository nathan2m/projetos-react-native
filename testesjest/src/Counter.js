import React, {Component} from 'react';

import {StyleSheet, Text, View} from 'react-native';

class Counter extends Component {
  render() {
    return (
      <View style={styles.container}>
        <Text>VocÃª tem {this.props.count * 2} counters.</Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignContent: 'center',
    alignItems: 'center',
    justifyContent: 'center',
  },
});

export default Counter;
