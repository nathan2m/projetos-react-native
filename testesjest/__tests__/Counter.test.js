import 'react-native';
import React from 'react';
import {shallow} from 'enzyme';
import Counter from '../src/Counter';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  /*  const tree =  */ renderer.create(<Counter count={5} />);
  //expect(tree).toMatchSnapshot();
});

/* describe('Testing App Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Counter count={5} />);

    expect(wrapper).toMatchSnapshot();
    wrapper.setProps({count: 10});
    expect(wrapper).toMatchSnapshot();
  });
}); */
