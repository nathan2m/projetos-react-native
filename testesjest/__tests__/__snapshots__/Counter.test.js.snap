// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Component should render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing App Component should render correctly 2`] = `ShallowWrapper {}`;
